apps/avifgainmaputil/extractgainmap_command.cc

```diff
@@ -18,7 +18,7 @@ ExtractGainMapCommand::ExtractGainMapCommand()
 
 avifResult ExtractGainMapCommand::Run() {
   DecoderPtr decoder(avifDecoderCreate());
-  if (decoder == NULL) {
+  if (decoder == nullptr) {
```

Comment:
Please move the NULL-to-nullptr changes to a separate pull request.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2742